runtime/vm/exceptionsupport.c

```diff
@@ -519,6 +519,16 @@ internalSetCurrentExceptionWithCause(J9VMThread *currentThread, UDATA exceptionN
 	currentThread->currentException = NULL;
 	currentThread->privateFlags &= ~J9_PRIVATE_FLAGS_REPORT_EXCEPTION_THROW;
 
+	if (J9VMCONSTANTPOOL_JAVALANGUNSUPPORTEDCLASSVERSIONERROR == index) {
```

Comment:
can you move this code to be closer to the `J9VMCONSTANTPOOL_JAVALANGSTACKOVERFLOWERROR` case below?

Reply:
Sure, moved it to be just before the case `(index == J9VMCONSTANTPOOL_JAVALANGSTACKOVERFLOWERROR)`. This grouped together different cases for `index`. It is in slow path anyway so "Preserve cause object" has no impact.
Please review.